docs/js-web-tools/bootstrap/bootstrap.rst

```diff
@@ -0,0 +1,241 @@
+
+.. code-block:: html
+
+	<button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+		<span class="icon-bar"></span>
+		<span class="icon-bar"></span>
+		<span class="icon-bar"></span>
+	</button>
+
+It also includes the application name, which appears in the top left.  The main navigation menu is rendered by the <ul> element within the second div, and includes links to Home, About, and Contact. Additional links for Register and Login are added by the _LoginPartial line on line 29.
+Below the navigation, the main body of each page is rendered in another <div>, marked with the "container" and "body-content" classes. In the simple default _Layout file shown here, the contents of the page are rendered by the specific View associated with the page, and then a simple <footer> is added to the end of the <div> element.  You can see how the built-in About page appears using this template:
+
+.. image:: _static/about-page-wide.png
+
+The collapsed navbar, with "hamburger" button in the top right, appears when the window drops below a certain width:
+
+.. image:: _static/about-page-hamburger.png
+
+Clicking the icon reveals the menu items in a vertical drawer that slides down from the top of the page:
+
+.. image:: _static/about-page-hamburger-open.png
+
+Typography and Links
+--------------------
+
+Bootstrap sets up the site's basic typography, colors, and link formatting in its CSS file. This CSS file includes default styles for tables, buttons, form elements, images, and more. One particularly useful feature is the grid layout system, covered next.
+
+Grids
+-----
+
+One of the most popular features of Bootstrap is its grid layout system. Modern web applications should avoid using the <table> tag for layout, instead restricting the use of this element to actual tabular data. Instead, columns and rows can be laid out using a series of <div> elements and the appropriate CSS classes. There are several advantages to this approach, including the ability to adjust the layout of grids to display vertically on narrow screens, such as on phones.
+
+Bootstrap's grid layout system is based on twelve columns. This number was chosen because it can be divided evenly into 1, 2, 3, or 4 columns, and column widths can vary to within 1/12th of the vertical width of the screen. To start using the grid layout system, you should begin with a container <div> and then add a row <div>, as shown here:
```

Comment:
Link to the bootstrap docs where applicable such as http://getbootstrap.com/css/#grid